tests/aws/services/events/test_events_rules.py

```diff
@@ -261,7 +263,8 @@ def test_put_event_with_content_base_rule_in_pattern(aws_client, clean_up):
             "test2": [{"anything-but": "test2"}],
             "test3": [{"anything-but": ["test3", "test33"]}],
             "test4": [{"anything-but": {"prefix": "test4"}}],
-            "ip": [{"cidr": "10.102.1.0/24"}],
```

Comment:
That IP check never worked but it got silently ignored 😬  (just double-checked by running the test suite against the old implementations before the refactorings and fixes)